Formula/couchdb.rb

```diff
@@ -58,6 +58,13 @@ def caveats
       If you want to migrate your data from 1.x to 2.x then follow this guide:
       https://docs.couchdb.org/en/stable/install/upgrading.html
```

Comment:
Can this (and the upgrade from 1.7.2_1 stuff above) be removed at this point?

Reply:
good call, I’ll ponder this a bit since there might be a portion of folks who are coming from 1.x and they will need different instructions. I’ll give those a go tomorrow.

Reply:
How long ago was 2.x released (it may help inform us here).

Reply:
September 20, 2016

Reply:
Yeah, that should be removed. If you didn't update your homebrew install in 4 years you're gonna have bigger issues then couchdb

Reply:
all done, thanks for the input!